rules/ml/ml_linux_anomalous_compiler_activity.toml

```
@@ -1,7 +1,9 @@
 [metadata]
 creation_date = "2020/09/03"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2022/03/16"
+min_stack_comments = "Supports latest version of ML job introduced in 8.2"
+min_stack_version = "8.2.0"
```

I saw that the jobs are getting pushed to 8.3.0, so we will need to update these to reflect. Then we will need to wait to merge until we cut the 8.2 branch (making main 8.3), which should be next week

```suggestion
min_stack_version = "8.3.0"
```